packages/components/src/components/buttons/buttonStyleUtils.ts

```diff
@@ -107,13 +108,14 @@ export const getVariantStyle = (variant: ButtonVariant, elevation: Elevation) =>
             `;
         case 'tertiary':
             return css`
-                background: ${({ theme }) => theme[mapElevationToBackgroundTertiary[elevation]]};
+                background: ${({ theme }) =>
+                    theme[mapElevationToBackgroundTertiaryNormal[elevation]]};
```

Comment:
I'm thinking about using some functions with easier names for these extra-long maps. WDYT?

Reply:
Good idea, I have the same plans, but its probably for another day.

Reply:
OK, just please create an issue for it in github 🙏